GCP_VM_Create.yml

```diff
@@ -9,36 +9,36 @@
     region: "europe-west3"
 
   tasks:
-   - name: create a disk
-     gcp_compute_disk:
-         name: "{{ vm_disk_name | default('ansible-tower-test-disk', true) }}"
-         size_gb: 50
-         source_image: 'projects/ubuntu-os-cloud/global/images/family/ubuntu-1604-lts'
-         zone: "{{ zone }}"
-         project: "{{ gcp_project }}"
-         state: present
-     register: disk
-   - name: create a address
-     gcp_compute_address:
-         name: "{{ vm_address_name | default('ansible-tower-test-address', true) }}"
-         region: "{{ region }}"
-         project: "{{ gcp_project }}"
-         state: present
-     register: address
-   - name: create a instance
-     gcp_compute_instance:
-         state: present
-         name: "{{ vm_name | default('ansible-tower-test', true) }}"
-         machine_type: n1-standard-1
```

Comment:
hardoced machine type

Reply:
+1 
Should be set as variable

Reply:
I will add empty line between tasks. Just to make code more clean and

Reply:
This is for the PoC part to demonstrate that we can run VM in GCP. I don't know if we want to parametrize machine type. I would keep it as default for no wand parametrize it once we define a set of values we want to be configurable for all clouds. 
WDYT?

Reply:
if you keep it hardoced then create an issue please. otherwise we can easily carry it.